portal/src/main/webapp/js/lib/mutationMapper.js

```diff
@@ -16020,6 +16061,24 @@ function MutationDetailsTable(options, gene, mutationUtil, dataProxies, dataMana
 		}
 	}
 
+	function uniqueSamples(rowData)
+	{
+		var samples = {};
+
+		_.each(rowData, function(data, index) {
+			// assuming only the first element contains the datum
+			var mutation = data[0].mutation;
```

Comment:
Actually assuming that we are passing an array of row data to this function. And the data is retrieved by this API function: https://datatables.net/reference/api/data()